docs/source/Tutorial.md

```diff
@@ -1913,8 +1911,7 @@ appropriate `finish` method.
   // Call `Finish()` to instruct the builder that this monster is complete.
   // Note: Regardless of how you created the `orc`, you still need to call
   // `Finish()` on the `FlatBufferBuilder`.
-  builder.Finish(orc); // You could also call `FinishMonsterBuffer(builder,
```

Comment:
How does this change help? seems harder to read on two lines. We don't have a strict 80 chars limit in the docs.